src/components/HomePage.vue

```diff
@@ -136,6 +142,22 @@
     toggleToDoEditState();
   }
 
+  function filterToDos(filterCategory: string, order: number) {
+    if (filterCategory === '') {
```

Comment:
Please change the condition to `!filterCategory`. It will be false if the filterCategory is undefined, null or an empty string as well.